restcomm/restcomm.extension.controller/src/main/java/org/restcomm/connect/extension/controller/ExtensionController.java

```diff
@@ -282,4 +304,29 @@ public ExtensionResponse executePostApiAction(final ApiRequest apiRequest, List<
         }
         return response;
     }
+
+    @Override
+    public ExtensionConfiguration getEffectiveConfiguration(String extensionSid, String scopeSid) {
+        ExtensionsConfigurationDao ecd = daoManager.getExtensionsConfigurationDao();
+        ClientsDao cd = daoManager.getClientsDao();
+        AccountsDao ad = daoManager.getAccountsDao();
+        OrganizationsDao od = daoManager.getOrganizationsDao();
+        ExtensionConfiguration extCfg = ecd.getAccountExtensionConfiguration(scopeSid, extensionSid);
+
+        Sid sid = new Sid(scopeSid);
```

Comment:
can we save cost of multiple queries here by querying the exact dao which sid belong to